### ISSUE TITLE
Details on code workings ?

### ISSUE THREAD

**Issue by VishalIstek** [tastelessjolt/youtube-dl-android#21]
It would be nice if you put comments on codes so we can understand how it is working and further we can help to add other sites ( which as you told your not going to add them soon to your project)
But many including me is waiting for the support of additional sites such as Hotstar Sony and other streaming sites.
Youtube-dl ofc have these but for compatibility, this would be great.
Waiting for your reply.

**Comment by xibr** [comment]
Yes I agree with you, we want to add **Twitter** and **Instagram**  and other sites, thank you very much
